ros/angel_system_nodes/setup.py

```diff
@@ -40,3 +42,4 @@
             ],
     },
 )
+1
```

Comment:
Is this an accidental "1" value at the end of the file?

Reply:
Will resolve.